components/studentPortal/layout/Sidebar.tsx

```diff
@@ -69,8 +85,11 @@ export const Sidebar: React.FC = () => {
     onCompleted: (data) => {
       if (data.users[0].userRole.value === "coach") {
         dispatch(setUserRole("coach"));
-      } else {
+      } else if (data.users[0].userRole.value === "student") {
         dispatch(setUserRole("student"));
+      } else if (data.users[0].userRole.value === "freemium") {
+        dispatch(setCreatedAt(data.users[0].created_at));
```

Comment:
won't this get set everytime a freemium user logs in?

Reply:
we can actually create a created_at column in the users table in hasura. and default the value to be `now()` this way, we will set that column, anytime a new user is created and added to the table.

Reply:
no. this is the date that the account was created at.  And it is a column in the users table, and I am adding it into this query and it should be set to now ()

Reply:
my mistake, i didn't understand what was happening. looks good tome